apps/documentation/src/routes/documentation/contributing/local-development.mdx

```diff
@@ -0,0 +1,111 @@
+import MetaTags from '../../../components/meta-tags'
+
+<MetaTags
+  title="Tuono - Contributing - Local development"
+  canonical="https://tuono.dev/documentation/contributing/local-development"
+  description="The project is massive - if you like it, do consider contributing!"
+/>
+
+import Breadcrumbs, { Element } from '../../../components/breadcrumbs'
+
+<Breadcrumbs breadcrumbs={[{ label: '✨ Contributing' }]} />
+
+# Local development
+
+Thanks for your interest! In this page you find the instructions to setup the `tuono`!
+
+## Quick setup
+
+```sh
+git clone https://github.com/<your-name-here>/tuono
+cd tuono
+pnpm i
+pnpm run check-all
+```
+
+## Setup
+
+### Fork and clone repository
+
+After [forking the repo on Github](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo):
+
+```sh
+git clone https://github.com/<your-name-here>/tuono
+cd tuono
+```
+
+### Rust tool chain
+
+Install the Rust programming language tool chain (`rust` and `cargo`).
+Follows instructions in the official [docs](https://rustup.rs/)
+
+### NodeJS - runtime
+
+Install `NodeJS`.
+You can follow the instructions from the [Node official site](https://nodejs.org/en/download/package-manager)
+
+> 💡 This project has a `.nvmrc` file to specify the node version used in development.
+>
+> Consider to use [nvm](https://github.com/nvm-sh/nvm) so you can run
+>
+> ```sh
+> nvm use
+> ```
+>
+> to simply pickup the correct version!
+
+### NodeJS - package manager
+
+We use [`pnpm`](https://pnpm.io) as `NodeJS` package manager.
+
+You can see which version of yarn we use by checking the `packageManager` field in the root `package.json`.
+
+### Pre-flight checks
+
+To check that everting is working correctly run:
+
+```sh
+turbo run check-all
+cargo build
+```
+
+## Tuono development
+
+1. Start tuono frontend build using
+
+   ```sh
+   turbo run dev
+   ```
+
+1. In another shell run
+
+   ```sh
+   cargo build
```

Comment:
We could mention to install `cargo-watch` crate that allows automatic re-builds with `cargo watch -x build -w crates/`